src/main/java/codeu/controller/ChatServlet.java

```diff
@@ -14,23 +14,23 @@
 
 package codeu.controller;
 
-import codeu.model.data.Conversation;
-import codeu.model.data.Message;
-import codeu.model.data.User;
+import codeu.model.data.*;
 import codeu.model.store.basic.ConversationStore;
 import codeu.model.store.basic.MessageStore;
 import codeu.model.store.basic.UserStore;
 import java.io.IOException;
 import java.time.Instant;
-import java.util.List;
-import java.util.UUID;
+import java.util.*;
 import javax.servlet.ServletException;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+
+import com.google.appengine.repackaged.com.google.common.collect.ImmutableList;
```

Comment:
Switch this class to com.google.common.collect.ImmutableList -- don't use the appengine.repackaged one (I think that's your IDE trying to be cool, when it really isn't).